pkg/vendors/mellanox/mellanox.go

```diff
@@ -144,10 +145,25 @@ func (m *mellanoxHelper) GetMellanoxBlueFieldMode(PciAddress string) (BlueFieldM
 	return -1, fmt.Errorf("MellanoxBlueFieldMode(): unknown device status for %s", PciAddress)
 }
 
-func (m *mellanoxHelper) MlxResetFW(pciAddresses []string) error {
+func (m *mellanoxHelper) MlxResetFW(pciAddresses []string, mellanoxNicsStatus map[string]map[string]sriovnetworkv1.InterfaceExt, hostHelper host.HostManagerInterface) error {
```

Comment:
can we just change the order here https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/1921bed36fa8d97e5c7989b2c02a67fcdaa587e2/pkg/helper/host.go#L29 and add the hostInterface to here?
```
func New(utilsHelper utils.CmdInterface) MellanoxInterface {
	return &mellanoxHelper{
		utils: utilsHelper,
	}
}
```